### ISSUE TITLE
Keyboard Hotkeys not working in Pantheon session

### ISSUE THREAD

**Issue by austinmarner** [decathorpe/elementary-staging-rpms#5]
I found that hotkeys are not working after upgrading to Fedora 30, I installed gnome to test but the hotkeys work perfectly in Gnome session but in pantheon session, it doesn't work

**Comment by austinmarner** [comment]
I heard that Pantheon apps and session dependencies are available by default in Fedora 30 repo, is it true?

If yes, then can I remove elementary-nightly and staging copr repos?

**Comment by decathorpe** [comment]
Yes, the COPR repositories haven't been necessary for more than a year or so

Keyboard shortcuts should work however. Which ones don't work? Can you try to open the keyboard settings in switchboard and see if changing the shortcuts there changes anything?

**Comment by decathorpe** [comment]
(Side note: removing or disabling the repos now won't do anything unless you also run `sudo dnf distro-sync` afterwards to downgrade to the latest stable versions)

**Comment by austinmarner** [comment]
none of the function hotkeys are working except keyboard backlit

all were working fine with pantheon in Fedora 29 but after upgrading to 30 it stopped working, and I tested it in gnome but in gnome, everything works perfectly

**Comment by decathorpe** [comment]
Ok, I will try to investigate

**Comment by austinmarner** [comment]
Any improvement?

**Comment by decathorpe** [comment]
Not yet, sorry. I've been super busy at University

**Comment by decathorpe** [comment]
The only thing I know is that this is probably caused by a change in the dbus interface exported by gnome-settings-daemon.

**Comment by worldofpeace** [comment]
Sounds like https://github.com/elementary/gala/issues/518

**Comment by austinmarner** [comment]
but not even normal shortcuts working, like super+t for the terminal, ctl+alt+t, etc
but normal shortcuts like ctrl+c and v are working fine

**Comment by austinmarner** [comment]
So that gnome-settings-daemon is the one currently having the bug?

**Comment by decathorpe** [comment]
If I understand correctly the issue is caused by gala not supporting gnome-settings-daemon version 3.32.

**Comment by javadr** [comment]
I have the same problem after upgrading Fedora 29 to 30! While all function keys work perfectly in gnome, none of them works in Pantheon.
